services/ui-waflog-s3-bucket/serverless.yml

```diff
@@ -0,0 +1,30 @@
+service: ui-waflog-s3-bucket
+
+frameworkVersion: "2"
+
+plugins:
+  - serverless-stack-termination-protection
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-1
+
+custom:
+  stage: ${opt:stage, self:provider.stage}
+  serverlessTerminationProtection:
+    stages:
+      - master
+      - val
+      - production
+
+resources:
+  Resources:
+    WaflogsUploadBucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: ${self:custom.stage}-waflogsbucket
```

Comment:
This is problematic, as S3 is a global name space.
I recommend using a random prefix, or the account id, at the start of the bucket name.
```BucketName:  ${AWS::AcountId}-${self:service.name}-${self:custom.stage}-waflogs```

Reply:
done